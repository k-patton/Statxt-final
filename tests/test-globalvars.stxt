int a;
float f;
char c;
bool b;
string s;
int[3] arr;

void printfunc() {
	print(a);
	printf(f);
	printchar(c);
	printb(b);
	printstr(s);
	print(arr[1]);
	
	int i;
	for(i = 0; i < 3; i++) {
		print(arr[i]);
	}
}


int main() {
	a = 1;
	f = 1.1;
	c = 'a';
	b = true;
	s = "supercalifragilistic expielidocious";
	arr = [4,5,6];

	printfunc();
}