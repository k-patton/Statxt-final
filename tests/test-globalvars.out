1
1.1
a
1
supercalifragilistic expielidocious
5
4
5
6
